Library/Homebrew/extend/array.rbi

```diff
@@ -0,0 +1,15 @@
+# typed: strict
+
+class Array
+  sig { returns(T.nilable(Elem)) }
```

Comment:
`Elem` is an [upstream](https://github.com/sorbet/sorbet/blob/5e26c51/rbi/core/array.rbi#L352) RBI creation that isn't available at runtime, so these need to be in RBI files. (And similarly for `Enumerable::Enum`, `Hash::K`, `Hash::V`, etc. below…)